DXRMissions/DeusEx/Classes/DXRMissionsM03.uc

```diff
@@ -53,7 +53,104 @@ function int InitGoals(int mission, string map)
         AddGoalLocation("03_NYC_AIRFIELD", "West Security Tower", NORMAL_GOAL, vect(-2060.626465, -2013.138672, 15.090023), rot(0, 0, 0));
         AddGoalLocation("03_NYC_AIRFIELD", "NW Security Tower", NORMAL_GOAL, vect(729.454651, -4151.924805, 15.079981), rot(0, 0, 0));
         AddGoalLocation("03_NYC_AIRFIELD", "NE Security Tower", NORMAL_GOAL, vect(5215.076660, -4134.674316, 15.090023), rot(0, 0, 0));
-        AddGoalLocation("03_NYC_AIRFIELD", "Hanger Door", NORMAL_GOAL, vect(941.941895, 283.418152, 15.090023), rot(0, 0, 0));
+        AddGoalLocation("03_NYC_AIRFIELD", "Hangar Door", NORMAL_GOAL, vect(941.941895, 283.418152, 15.090023), rot(0, 0, 0));
```

Comment:
git diff, plz

Reply:
"Hanger" to "Hangar"

Reply:
the whole diff is ruined, not just that line lol